DeeployTest/testUtils/codeGenerate.py

```
@@ -97,6 +100,11 @@ def generateTestInputsHeader(deployer: NetworkDeployer, test_inputs: List, input
     ])
     retStr += "};\n"
 
+    if verbose:
+        print('Input:')
+        for name, buf, offset in zip(inputTypes.keys(), inputNames, inputOffsets.values()):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)